governance-and-automation/security-compliance-checks/private-workspaces-shared-with-everyone.md

```diff
@@ -8,24 +8,28 @@ Syskit Point detects **private workspaces shared with Everyone, Everyone except
 
 Oversharing workspaces might lead to **security risks, administration difficulty, and a challenge managing access control**.
 
-The Private Workspaces Shared with Everyone is **a tenant-wide policy, and it cannot have task delegation enabled**. That means no tasks are created to resolve this policy, and no emails are sent to collaborators.
+The Private Workspaces Shared with Everyone is **a tenant-wide policy and can be automated without task delegation**. That means no tasks are created to resolve this policy, and no emails are sent to collaborators. 
 
-The private workspaces shared with everyone **are still detected** as a part of the Security and Compliance checks feature. **This means that Syskit Point detects a vulnerability on a workspace** based on the applied policy, but it does not create tasks or send any emails to collaborators. 
+By default, the Private Workspaces Shared with Everyone policy is set to **Resolve Manually**, which means the vulnerability is flagged on the Security and Compliance Dashboard, where manual actions can be taken to resolve it. 
+
+If the policy is set to **Resolve Automatically**, Syskit Point **automatically resolves the vulnerability** by removing access granted to Everyone, Everyone except external users, and All users groups. 
+
+The private workspaces shared with everyone **are still detected** as a part of the Security and Compliance checks feature. **This means that Syskit Point detects a vulnerability in a workspace** based on the applied policy, but it does not create tasks or send any emails to collaborators. 
```

Comment:
za ovu rečenicu mi sad nije jasno na što se točno odnosi - dal na resolve manually option ili resolve automatically option